Content.Server/Administration/Systems/BwoinkSystem.cs

```diff
@@ -87,14 +98,128 @@ private void OnOverrideChanged(string obj)
+                var ban = await _dbManager.GetServerBanAsync(null, e.Session.UserId, null);
+                if (ban != null)
+                {
+                    var banMessage = Loc.GetString("bwoink-system-player-banned", ("banReason", ban.Reason));
+                    NotifyAdmins(e.Session, banMessage, PlayerStatusType.Banned);
+                    _activeConversations.Remove(e.Session.UserId);
+                    return;
+                }
+            }
+
+            // Notify all admins if a player disconnects or reconnects
+            var message = e.NewStatus switch
+            {
+                SessionStatus.Connected => Loc.GetString("bwoink-system-player-reconnecting"),
+                SessionStatus.Disconnected => Loc.GetString("bwoink-system-player-disconnecting"),
+                _ => null
+            };
+
+            if (message != null)
+            {
+                var statusType = e.NewStatus == SessionStatus.Connected ? PlayerStatusType.Connected : PlayerStatusType.Disconnected;
+                NotifyAdmins(e.Session, message, statusType);
+            }
+
             if (e.NewStatus != SessionStatus.InGame)
                 return;
 
             RaiseNetworkEvent(new BwoinkDiscordRelayUpdated(!string.IsNullOrWhiteSpace(_webhookUrl)), e.Session);
         }
 
+        private void NotifyAdmins(ICommonSession session, string message, PlayerStatusType statusType)
+        {
+            if (!_activeConversations.ContainsKey(session.UserId))
+            {
+                // If the user is not part of an active conversation, do not notify admins.
+                return;
+            }
+
+            // Get the current timestamp
+            var timestamp = DateTime.Now.ToString("HH:mm:ss");
+            var roundTime = _gameTicker.RoundDuration().ToString("hh\\:mm\\:ss");
+
+            // Determine the icon based on the status type
+            string icon = statusType switch
+            {
+                PlayerStatusType.Connected => ":green_circle:",
+                PlayerStatusType.Disconnected => ":red_circle:",
+                PlayerStatusType.Banned => ":no_entry:",
+                _ => ":question:"
+            };
+
+            // Create the message parameters for Discord
+            var messageParams = new AHelpMessageParams(
+                session.Name,
+                message,
+                true,
+                roundTime,
+                _gameTicker.RunLevel,
+                playedSound: true,
+                icon: icon
+            );
+
+            // Create the message for in-game with username
+            var color = statusType switch
+            {
+                PlayerStatusType.Connected => "green",
+                PlayerStatusType.Disconnected => "yellow",
+                PlayerStatusType.Banned => "orange",
+                _ => "gray"
+            };
+            var inGameMessage = $"[color={color}]{session.Name} {message}[/color]";
```

Comment:
Localize the wrap

Reply:
I don't really understand this one, None of these need to be localized?

They are just passing through what ever the playername/message is. + the color is an actual color not the word so nothing to translate?